<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="xpitax-11568" failures="10" tests="31" name="Test" time="29.032" errors="0" timestamp="25 Mar 2021 15:03:58 GMT">
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="@BeforeClass deleteEmployee" time="1.728">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
at com.xcm.testcases.TC05_DeleteEmplyeRecord.deleteEmployee(TC05_DeleteEmplyeRecord.java:52)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:83)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:117)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
... 30 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass deleteEmployee -->
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="@AfterClass tearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="com.xcm.testcases.TC01_GetAllEmployeeData" name="checkContentType" time="0.028">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.xcm.testcases.TC01_GetAllEmployeeData.checkContentType(TC01_GetAllEmployeeData.java:108)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.xcm.testcases.TC01_GetAllEmployeeData" name="checkResponseBody" time="0.031"/>
  <testcase classname="com.xcm.testcases.TC01_GetAllEmployeeData" name="checkResponseTime" time="0.005"/>
  <testcase classname="com.xcm.testcases.TC01_GetAllEmployeeData" name="checkStatusCode" time="0.004"/>
  <testcase classname="com.xcm.testcases.TC01_GetAllEmployeeData" name="checkcontentEncoding" time="0.003"/>
  <testcase classname="com.xcm.testcases.TC01_GetAllEmployeeData" name="checkstatusLine" time="0.008"/>
  <testcase classname="com.xcm.testcases.TC02_GetSingleEmploye" name="checkContentLenght" time="0.005"/>
  <testcase classname="com.xcm.testcases.TC02_GetSingleEmploye" name="checkContentType" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.xcm.testcases.TC02_GetSingleEmploye.checkContentType(TC02_GetSingleEmploye.java:83)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.xcm.testcases.TC02_GetSingleEmploye" name="checkResponseTime" time="0.017"/>
  <testcase classname="com.xcm.testcases.TC02_GetSingleEmploye" name="checkResposeBody" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.xcm.testcases.TC02_GetSingleEmploye.checkResposeBody(TC02_GetSingleEmploye.java:53)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="com.xcm.testcases.TC02_GetSingleEmploye" name="checkStatusCode" time="0.009"/>
  <testcase classname="com.xcm.testcases.TC02_GetSingleEmploye" name="checkserverType" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx/1.16.0]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.16.0]
at com.xcm.testcases.TC02_GetSingleEmploye.checkserverType(TC02_GetSingleEmploye.java:90)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.xcm.testcases.TC02_GetSingleEmploye" name="checkstatusLine" time="0.002"/>
  <testcase classname="com.xcm.testcases.TC03_PostEmployeRecord" name="checkContentType" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.xcm.testcases.TC03_PostEmployeRecord.checkContentType(TC03_PostEmployeRecord.java:99)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.xcm.testcases.TC03_PostEmployeRecord" name="checkResposeBody" time="0.006"/>
  <testcase classname="com.xcm.testcases.TC03_PostEmployeRecord" name="checkStatusCode" time="0.004"/>
  <testcase classname="com.xcm.testcases.TC03_PostEmployeRecord" name="checkcontentEncoding" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [gzip] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at com.xcm.testcases.TC03_PostEmployeRecord.checkcontentEncoding(TC03_PostEmployeRecord.java:113)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkcontentEncoding -->
  <testcase classname="com.xcm.testcases.TC03_PostEmployeRecord" name="checkserverType" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx/1.16.0]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.16.0]
at com.xcm.testcases.TC03_PostEmployeRecord.checkserverType(TC03_PostEmployeRecord.java:106)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.xcm.testcases.TC03_PostEmployeRecord" name="checkstatusLine" time="0.002"/>
  <testcase classname="com.xcm.testcases.TC04_PutEmployeRecord" name="checkContentType" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.xcm.testcases.TC04_PutEmployeRecord.checkContentType(TC04_PutEmployeRecord.java:100)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.xcm.testcases.TC04_PutEmployeRecord" name="checkResposeBody" time="0.003"/>
  <testcase classname="com.xcm.testcases.TC04_PutEmployeRecord" name="checkStatusCode" time="0.004"/>
  <testcase classname="com.xcm.testcases.TC04_PutEmployeRecord" name="checkcontentEncoding" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [gzip] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at com.xcm.testcases.TC04_PutEmployeRecord.checkcontentEncoding(TC04_PutEmployeRecord.java:114)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkcontentEncoding -->
  <testcase classname="com.xcm.testcases.TC04_PutEmployeRecord" name="checkserverType" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx/1.16.0]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.16.0]
at com.xcm.testcases.TC04_PutEmployeRecord.checkserverType(TC04_PutEmployeRecord.java:107)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.xcm.testcases.TC04_PutEmployeRecord" name="checkstatusLine" time="0.003"/>
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="checkContentType" time="0.0">
    <skipped/>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="checkResposeBody" time="0.0">
    <skipped/>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="checkStatusCode" time="0.0">
    <skipped/>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="checkcontentEncoding" time="0.0">
    <skipped/>
  </testcase> <!-- checkcontentEncoding -->
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="checkserverType" time="0.0">
    <skipped/>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.xcm.testcases.TC05_DeleteEmplyeRecord" name="checkstatusLine" time="0.0">
    <skipped/>
  </testcase> <!-- checkstatusLine -->
</testsuite> <!-- Test -->
